src/worker.browser/worker.js

```diff
@@ -18,8 +18,12 @@ function joinPaths (path1, path2) {
 }
 
 function prependScriptUrl(scriptUrl) {
-  const prefix = getConfig().basepath.web;
-  return prefix ? joinPaths(prefix, scriptUrl) : scriptUrl;
+  let basePath = getConfig().basepath.web;
+  if (basePath && !basePath.match(/^\w+:\/\//)) {
+    // Turn path into full URL, since referencing a `http://` path from a `blob://` URL won't work
+    basePath = `${document.location.protocol}//${document.location.host}${basePath}`;
```

Comment:
Need to check: Does `document.location.protocol` equal `http:`|`https:`, or is it `blob:`?